sql_queries/Flag_ExactDuplicates.sql

```diff
@@ -0,0 +1,15 @@
+﻿SET SEARCH_PATH = marvl3, public;
+
+-- Identify XBT duplicates
+UPDATE spatial_subset SET duplicate = TRUE
```

Comment:
We also need to increment a duplicate_id. Declare a sequence when creating spatial subset table, like:
CREATE SEQUENCE duplicate_seq START 0;

And then you can retrieve incremental values of the sequence by doing:
SELECT nextval('duplicate_seq');

So at this line you may have something like:
UPDATE spatial_subset SET duplicate = TRUE, duplicate_id = (SELECT nextval('duplicate_seq'))